README.md

```diff
@@ -1,177 +1,164 @@
 
-![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/web-console.png)
+    ![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/web-console.png)
 
-We can see the RedHat OpenShift Container Platform (Web Console). Click on question mark icon on the top right hand side and select `Command Line Tools`. 
+1. You should see the RedHat OpenShift Container Platform web console. Click the question mark icon on the top right-hand side and select **Command Line Tools**. 
 
-![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/cmd-line-tools.png)
+    ![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/cmd-line-tools.png)
 
-Navigate to the section `oc - OpenShift Command Line Interface (CLI)` and download the respective oc binary onto your local system. This is needed to manage OpenShift projects from a terminal and is further extended to natively support OpenShift Container Platform features.
+1. Navigate to the section `oc - OpenShift Command Line Interface (CLI)` and download the respective oc binary onto your local system. You need this to manage OpenShift projects from a terminal. The binary is extended to natively support OpenShift Container Platform features.
 
-![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/oc-binary.png)
+    ![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/oc-binary.png)
 
-We are all set to proceed to next step which is to register the OpenShift cluster on Red Hat Marketplace platform. This is mandatory to install any operators from RedHat Marketplace platform using the OpenShift cluster.
+Now you are ready to register the OpenShift cluster on Red Hat Marketplace. This step is mandatory to install any operators from Red Hat Marketplace platform using the OpenShift cluster.
 
-### Register the cluster on Red Hat Marketplace
+## Register the cluster on Red Hat Marketplace
 
-Sign up and login to RHM portal at [Link](https://marketplace.redhat.com/en-us) and click on workspace and then click on cluster. We need to add our new OpenShift cluster and register it on RHM platform.
+1. Log into the [Red Hat Marketplace](https://marketplace.redhat.com/en-us). Select a workspace and click **Cluster**. You need to add the new OpenShift cluster and register it on the Red Hat Marketplace platform.
 
-![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/add-cluster.png)
+    ![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/add-cluster.png)
 
-Update the cluster name, generate the pull secret as per the instructions and save it. 
+1. Update the cluster name, generate the pull secret as per the instructions and save it. <!--EM: I'm confused about this. On the screen it looks like you create the secret, install the operator and then lick "Add cluster" but that's not what you're describing here is it?-->
 
```

Comment:
Click on add cluster and then give it a name, generate pull secret and save it. The image and steps are describing the same. Typo - lick <> click.